### PR TITLE
Added PTQ section to BERT notebook

### PR DESCRIPTION
**Pull request by svenchilton**
# Description

Added a section on post-training quantization to the BERT notebook. This follows @peri044's fixes in https://github.com/pytorch/TensorRT/pull/1191 to allow support for post-training quantization with multiple inputs. 

Fixes # (issue)

## Type of change

- New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines of this project (You can use the linters)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas and hacks
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests to verify my fix or my feature
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have added the relevant labels to my PR in so that relevant reviewers are notified

### CODE REVIEW

**Comment by svenchilton** [comment]
@narendasan, the github-actions bot says that some C++ code doesn't conform to the style guidelines and is requesting that I make some changes. I haven't touched the C++ code.

**Comment by svenchilton** [comment]
@peri044, the github-actions bot has once again requested that I make changes to some C++ code which it claims doesn't conform to the style guidelines. I haven't touched the C++ code.

**Comment by narendasan** [comment]
> @peri044, the github-actions bot has once again requested that I make changes to some C++ code which it claims doesn't conform to the style guidelines. I haven't touched the C++ code.

Don't worry about it, it is talking about some standing issues that got checked in which will be resolved in an other PR

File: py/torch_tensorrt/ptq.py [hunk 1]
```diff
@@ -30,12 +30,13 @@ def get_batch(self, names):
 
     batch = self.dataset_iterator.next()
     self.current_batch_idx += self.batch_size
-    # Treat the first element as input and others as targets.
+    inputs_gpu=[]
```
[the remark this answers is not in the document]
Comment (svenchilton) [reply py/torch_tensorrt/ptq.py:33]: @narendasan, sorry about the delayed reply. I think those changes are the very same ones that @peri044 checked in earlier.
